__tests__/integration/one-app.spec.js

```diff
@@ -873,6 +873,29 @@ describe('Tests that can run against either local Docker setup or remote One App
         });
       });
 
+      describe('code-splitting', () => {
+        test('successfully loads a code-split module with a language pack', async () => {
+          await browser.url(`${appInstanceUrls.browserUrl}/demo/franks-burgers`);
+          const openerMessage = await browser.$('#franks-opening-line');
+          await openerMessage.waitForExist();
+          expect(await openerMessage.getText()).toBe(
+            'Welcome to Franks Burgers! The best burgers in town.'
+          );
+        });
+
+        test('loads up the Order chunk and then lazy loads Calendar chunk with "react-calendar" on next', async () => {
+          await browser.url(`${appInstanceUrls.browserUrl}/demo/franks-burgers`);
+
+          const btn = await browser.$('#order-burger-btn');
+          await btn.waitForExist();
+          await btn.click();
+          await waitFor(1e3);
```

Comment:
Instead of `waitFor` what about `await franksBurger.waitForExist()`?

Reply:
Good suggestion, will update

Reply:
updated in cb5e958